Exif IPTC/5.8.0/metadata/asset/images.dcl

```diff
@@ -0,0 +1,5 @@
+resource meta_group_folder images {
+    parent_id = data.meta_group_folder.asset.id
+    name = 'Images'
+}
```

Comment:
I think the `Mobile` layer has a resource that is identical to this one, which might lead to issues if both the `Mobile` layer and this layer are needed simultaneously.

In the `Mobile` layer, only the `Orientation` field is included, which is also included in this layer.

I think we should address this - possible by removing it from the `Mobile` layer?

Reply:
I don't know of any customers that actually has the mobile layer applied, so yes, i think we should remote it at some point. Can we ignore the issue for now however?

Reply:
Sure 👍